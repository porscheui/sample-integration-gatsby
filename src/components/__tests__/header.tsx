import React from 'react';
import renderer from 'react-test-renderer';

import Header from '../header';

describe('Header', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Header siteTitle="Porsche Design System in Gatsby" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
