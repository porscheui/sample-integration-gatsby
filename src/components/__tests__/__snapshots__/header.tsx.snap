// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<p-grid>
  <p-grid-item
    size={12}
  >
    <p-headline
      align="center"
      variant="headline-2"
    >
      Porsche Design System in Gatsby
    </p-headline>
  </p-grid-item>
  <p-grid-item
    size={12}
  >
    <p-divider />
  </p-grid-item>
  <p-grid-item
    size={12}
  >
    <a
      className="removeLinkStyle"
      href="/collectionPage"
    >
      <p-link-pure>
        Components Collection
      </p-link-pure>
    </a>
    <a
      className="removeLinkStyle"
      href="/formsPage"
    >
      <p-link-pure>
        Forms
      </p-link-pure>
    </a>
  </p-grid-item>
  <p-grid-item
    size={12}
  >
    <p-divider />
  </p-grid-item>
</p-grid>
`;
